### PR TITLE
Add Hackathon blog post

### PR DESCRIPTION
**Pull request by cgebken**

### CODE REVIEW

File: _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: "A time travel through our latest Hackathon"
+date: 2019-04-01
+header_image: public/loc-interview.jpg
```
Comment (cgebken) [inline _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md]: Björn promised to have a look but he needed to travel back to Jena. Will add this on Monday.

File: _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+Now, it was time to decide which topic we'd like to join.
+Let me tell you, it was a tough decision.
+So many interesting but different topics.
+Some of them aimed to improve our processes and workflows, such as a standardized on-boarding process, unified PR pipelines, a performance test, or a Circle CI wallboard.
+Others had a closer look at our products and merchants, e.g. a UX check for integrating apps, an app for importing products via CSV, or an instore checkout.
+
+These questions helped me to decide which topic I'd like to join:
+- How can I support the team that is working on that topic?
+- What will I learn when participating?
+- Which team members will I have?
```
Comment (BirgitBader) [inline _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md]: Whom would I work with?

File: _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+So many interesting but different topics.
+Some of them aimed to improve our processes and workflows, such as a standardized on-boarding process, unified PR pipelines, a performance test, or a Circle CI wallboard.
+Others had a closer look at our products and merchants, e.g. a UX check for integrating apps, an app for importing products via CSV, or an instore checkout.
+
+These questions helped me to decide which topic I'd like to join:
+- How can I support the team that is working on that topic?
+- What will I learn when participating?
+- Which team members will I have? 
+
+Especially the last point was one of the most important ones for me.
```
Comment (BirgitBader) [inline _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md]: Hm, don't know if that's a bit toooo people-picky :D ?
Maybe then better combine these two sentences "as a hackathon is the "

File: _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+A Hackathon is the perfect chance to work together with colleagues that I don't work with on a daily basis.
```
Comment (BirgitBader) [inline _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md]: hackathon

File: _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+Maybe even with colleagues from different locations.
+This way, we can even build little "feature teams" for the two days.
```
Comment (BirgitBader) [inline _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md]: ... we can build small...

File: _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+As Birgit covered the About us page, I took the chance to join a group which had the goal to categorize our merchants and their shops.
```
Comment (BirgitBader) [inline _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md]: As my team mate Birgit took the lead for the "About us page" group, I joined a team that had...

File: _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+The results of such a categorization could help Product Management to make decisions on future features or improvements.
+And of course, I also had my personal goal: Get some improvement ideas for our help articles.
```
Comment (BirgitBader) [inline _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md]: ...: get some ideas to improve our help articles.

File: _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+## Last Wednesday
```
Comment (BirgitBader) [inline _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md]: Fist hack day

File: _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+
+The time has come.
+It's Hackathon!
```
Comment (BirgitBader) [inline _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md]: hackathon

File: _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+And what could be the best way to start the first day?
+Right - Breakfast!
```
Comment (BirgitBader) [inline _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md]: Right! By having a breakfast!

File: _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+
+The time has come.
+It's Hackathon!
+And what could be the best way to start the first day?
+Right - Breakfast! 
+
+{% twitter https://twitter.com/epagesdevs/status/1110812533349322752 %}
+
+The room was packed with ePagees.
+Our colleagues from Jena, Barcelona, and Bilbao already arrived the day before and everyone was chatting... and eating.
```
Comment (BirgitBader) [inline _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md]: ..., and everyone...

File: _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+Around 10 am the last buns and bagels disappeared, and our CTO officially kicked off the event.
+
+During the rest of the day, everyone was diving into their topics, brainstorming, discussing, hacking, and prototyping.
```
Comment (BirgitBader) [inline _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md]: For the rest of the day, each team was diving into its topic, brainstorming, discussing, hacking, or developing prototypes.

File: _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+When walking across the floor, I've seen many colored white boards and concentrated faces.
+
+Around 6 pm one word was spread across the floor... PIZZA.
+In no time, all teams came together and enjoyed the food.
+While eating, I could already notice that many colleagues fell in love with their topics.
+They were quite satisfied with their results and had big plans for the next day... or even the night.
```
Comment (BirgitBader) [inline _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md]: ..., and had...

File: _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md [hunk 14]
```diff
@@ -0,0 +1,102 @@
...
+
+{% twitter https://twitter.com/jensfischerhh/status/1110984432444456961 %}
+
+## Last Thursday
```
Comment (BirgitBader) [inline _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md]: Second hack day

File: _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md [hunk 15]
```diff
@@ -0,0 +1,102 @@
...
+
+Did I mention that we'll get Amazon vouchers for the first three groups?
```
Comment (BirgitBader) [inline _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md]: Did I mention that the winners of the hackathon get an Amazon voucher (and also the 2nd and 3rd place)?

File: _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md [hunk 16]
```diff
@@ -0,0 +1,102 @@
...
+No?
+Well, now you'll understand that after so many satisfied colleagues on the last evening, my group was even more motivated to collect great results.
```
Comment (BirgitBader) [inline _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md]: You make a switch here from a general description to your group.
It's a bit hard to understand.
If you wanna keep that, I suggest to start like this: 
Wann know what happened to my team? 

And then explain a bit more.
(Although I don't really know what you're exactly referring to in line 80)

File: _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md [hunk 17]
```diff
@@ -0,0 +1,102 @@
...
+So, we continued hacking, discussing, and analyzing.
+We got used to the tools, were able to improve our scripts, and shared our knowledge within our team.
+And of course, we remembered that each team has a presentation slot at the end of the day.
+No way, we prepared the slides in the last minutes because we were so into our tools and scripts.
+
+At 4:30 pm all teams gathered into one of our biggest rooms.
```
Comment (BirgitBader) [inline _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md]: ... into our biggest conference room.

File: _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md [hunk 18]
```diff
@@ -0,0 +1,102 @@
...
+It was time to share the results.
+As we were nine groups, we had a tough time schedule:
+Three minutes setup, five minutes presentation, and two minutes for questions.
+Afterwards, everyone had three votes.
+
+{% twitter https://twitter.com/epagesdevs/status/1111315953197948929 %}
+
+And the winner is... well, it would be a nice romantic-hollywood-movie-like ending for this post to tell you that my team won.
+But we didn't 😉.
+Instead, the standardized on-boarding process won, followed by our About us page (wohooo) and the instore checkout for our Beyond product.
```
Comment (BirgitBader) [inline _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md]: ..., as well as the Instore Checkout...

File: _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md [hunk 19]
```diff
@@ -0,0 +1,102 @@
...
+My team member Birgit and I also brainstormed and added a topic to the list: Improving the [About us page](/about/) of the DevPortal.
+
+## Two weeks ago
+
+All topics were set and added to the list.
+Now, it was time to decide which topic we'd like to join.
+Let me tell you, it was a tough decision.
+So many interesting but different topics.
+Some of them aimed to improve our processes and workflows, such as a standardized on-boarding process, unified PR pipelines, a performance test, or a Circle CI wallboard.
+Others had a closer look at our products and merchants, e.g. a UX check for integrating apps, an app for importing products via CSV, or an instore checkout.
```
Comment (BirgitBader) [inline _posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md]: Instore Checkout

**Pull request merged**